tests/helpers.py

```diff
@@ -46,6 +46,7 @@
 Redis Environment
 """
 testenv["redis_host"] = os.environ.get("REDIS_HOST", "127.0.0.1")
+testenv["redis_db"] = os.environ.get("REDIS_DB", 1)
```

Comment:
Why is the default value `1`?

Reply:
Redis has 16 databases by default. Default db was 0 by default before, but after I changed line 34 below, python takes 0 as none in the if block and code is not coming into if db block. I could use str(db) as well but I though using db 1 will be a better option

```python
db = ckw.get("db", None)

if host:
    url = f"redis://{host}:{port}"
    if db:
        url = f"{url}/{db}"
    span.set_attribute("connection", url)
```

Reply:
@CagriYonca , I think changing the db var to a string would be better.

Reply:
Good advice